### DISCUSSION TITLE
flexible modify for tags

### DISCUSSION THREAD

**Discussion by dosoe** [beetbox/beets#4424] [Q&A]
Hi! 
I'm trying to modify all the ```$title``` fields into ```$disc-$track $title```, which means that the ```title``` tag itself is modified. Is there a simple way to do it with ```beet modify``` (or any other way)? I tried some stuff, like ```beet modify singleton::False title=$disc-$track $title``` and variations thereof, but I couldn't get it to work properly. Is there a simple way to do this? I've seen the 'rewrite' plugin, but it doesn't change the tags itself. 
Thanks, 
Dorian

**Comment by sampsyo** [comment]
We actually have an in-flight pull request that would add this in https://github.com/beetbox/beets/pull/4095. Any help reviewing that to get it merged would be greatly appreciated!
